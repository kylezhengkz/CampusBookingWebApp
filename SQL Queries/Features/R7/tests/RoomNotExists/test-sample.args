{
    "args": [
        "00000000-0000-0000-0000-000000000004",
        "deaddead-dead-dead-dead-deaddeaddead",
        "2021-08-03 08:30:05",
        "2021-08-04 09:30:19",
        3
    ],
    "kwargs": {}
}   