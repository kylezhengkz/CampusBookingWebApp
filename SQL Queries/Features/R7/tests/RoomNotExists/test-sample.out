Booking failed: room is already booked.